pageserver/src/tenant/timeline.rs

```diff
@@ -1358,6 +1360,7 @@ impl Timeline {
 
                 compaction_lock: tokio::sync::Mutex::default(),
                 gc_lock: tokio::sync::Mutex::default(),
+                standby_flush_lsn: AtomicLsn::new(0),
```

Comment:
using Lsn::INVALID would make it more obvious that thise is the same as the default value in SkTimelineInfo

Reply:
`AtomicLsn` constructor accepts u64 parameter. And `AtomicLsn::new(LSN::INVALID.0)` looks IMHO really strange